packages/components/src/List/Toolbar/SelectSortBy/SelectSortBy.component.js

```diff
@@ -16,18 +36,7 @@ function SelectSortBy({ field, id, isDescending, onChange, options }) {
 		return onChange(event, { field: selected.id, isDescending: !order });
 	}
 
-	function getMenuItem(option, index) {
-		return (
-			<MenuItem
-				id={id && `${id}-by-item-${option.id}`}
-				key={index}
-				eventKey={option}
-			>
-				{option.name || option.id}
-			</MenuItem>
-		);
-	}
-
+	const getMenuItem = SortByItem;
```

Comment:
note: https://github.com/Talend/ui/pull/452/commits/15ab072bee721de764c80f853ccb4127517a14d0
if you call it using JSX you loose the onKeyDown ...